components/Card.tsx

```diff
@@ -7,9 +7,29 @@ interface CardProps {
   icon?: string;
   link?: string;
   image?: string;
+  headerSize?: 'small' | 'medium' | 'large';
+  bodyTextSize?: 'small' | 'medium' | 'large';
 }
 
-const CardBody = ({ title, body, icon, link, image }: CardProps) => {
+const CardBody = ({
+  title,
+  body,
+  icon,
+  link,
+  image,
+  headerSize,
+  bodyTextSize,
+}: CardProps) => {
+  const headerSizeClasses: Record<string, string> = {
+    small: 'text-[0.9rem]',
+    medium: 'text-[1.3rem]',
+    large: 'text-[2rem]',
+  };
+  const bodyTextSizeClasses: Record<string, string> = {
+    small: 'text-[0.85rem]',
+    medium: 'text-[1rem]',
+    large: 'text-[1.5rem]',
+  };
   return (
     <div className='group relative h-full w-full max-w-lg rounded-lg border border-gray-200 bg-white p-6 px-12 shadow-3xl transition-colors delay-[150ms] ease-in-out hover:bg-slate-100'>
```

Comment:
Here you can add the class `dark:text-black` to make the text inside cards dark-theme compatible